packages/react/src/patterns/blocks/ContentBlockMixed/__stories__/ContentBlockMixed.stories.js

```diff
@@ -17,13 +17,24 @@ storiesOf('Patterns (Blocks)|ContentBlockMixed', module)
   })
   .add('Default', () => {
     const heading = 'Lorem ipsum dolor sit amet';
+    const ctaStyles = {
+      text: 'text',
+      card: 'card',
```

Comment:
I think we only allow card cta style for the contentblockmixed pattern

https://ibm.ent.box.com/file/597369069325

Reply:
Was able to adjust via PR suggestions